RULES.md

```diff
@@ -57,6 +57,7 @@ Some are applicable for different technologies.
 | EC78     | Const parameter in batch update                               | Don't set const parameter in batch update => Put its in query. Creating this parameter and destroying it consumes CPU cycles and RAM unnecessarily.                                                                                                                                                                                                                                                                          |                                                                                                                                                                         | ✅   | 🚫  | 🚫 | 🚫    | 🚫   | 🚫 |
 | EC79     | Free resources                                                | try-with-resources Statement needs to be implemented for any object that implements the AutoCloseable interface, it save computer resources.                                                                                                                                                                                                                                                                                 |                                                                                                                                                                         | ✅   | 🚫  | 🚫 | 🚫    | 🚫   | 🚫 |
 | EC81     | Specify struct layouts                                        | When possible, specify struct layouts to optimize their memory footprint                                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                         | 🚫   | 🚫  | 🚫 | 🚫    | 🚫   | ✅ |
+| EC82     | Make variable constant                                        | A variable is never reassigned and can be made constant.                                                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                         | 🚀   | 🚀  | 🚀 | 🚀    | 🚀   | ✅ |
```

Comment:
it looks like you added rockets 🚀 to the EC203 rule (that was 🚫 everywhere before)
And on the new rule EC82, everything is 🚫  excepted the last column.
Is it normal ?

Reply:
EC203 is unchanged, on line 61 now.
EC82 has 🚀 everywhere because it should be applicable in every language, except for C# that has ✅ because the code is ready, just waiting for this PR validation first.